oak-run/src/test/java/org/apache/jackrabbit/oak/index/DocumentStoreIndexerIT.java

```diff
@@ -380,6 +387,95 @@ protected CompositeIndexer prepareIndexers(NodeStore nodeStore, NodeBuilder buil
 
     }
 
+    @Test
+    public void metrics() throws Exception {
```

Comment:
From what I can see, this test has a very similar setup as the bundling() test. Could you refactor that and create a setup function? Then it would be easier to know what the setup is and what is being tested.

Reply:
It's not so simple to share the common code because it is creating several objects that we need during the test. In this case, I think it is not worth the effort to improve the code, some duplication is not a big issue here as this is testing code and the duplication is contained in the same file.